Controller/Payment/Order.php

```diff
@@ -44,6 +45,14 @@ public function execute()
 
         $receipt_id = $this->getQuote()->getId();
 
+        $payment_action = $this->config->getPaymentAction();
+
+        if($payment_action == "authorize") {
```

Comment:
@seherkdc please update the following
- Space after `if`
- strict comparison
- single quotes around string
- brackets should be in new line

Reply:
@ccrims0n Changes Done.